<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>410</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>697</width>
    <height>360</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>697</width>
    <height>410</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Auto Cave Bot</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../assets/icons/Dead_Human3.gif</normaloff>../assets/icons/Dead_Human3.gif</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="bt_adicionar_ac">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Adicionar
Movimento</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_rm_ac">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>60</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Remover Ação</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_sleep">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>90</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size 10px</string>
    </property>
    <property name="text">
     <string>Add Sleep</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_script">
    <property name="geometry">
     <rect>
      <x>559</x>
      <y>120</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Script</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>11</y>
      <width>531</width>
      <height>351</height>
     </rect>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="sb_sleep">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>90</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_map_area">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>160</y>
      <width>24</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_autoloot">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>190</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto Loot</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_autoattack">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>220</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Auto attack</string>
    </property>
   </widget>
   <widget class="QToolButton" name="bt_autoloot">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>190</y>
      <width>24</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="bt_autoattack">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>220</y>
      <width>24</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_bot">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>160</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto Cave</string>
    </property>
   </widget>
   <widget class="QKeySequenceEdit" name="keySequenceEdit">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>270</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="keySequence">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>250</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Spells enquanto ataca:</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>697</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Importar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Exportar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>Novo</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
